src/axom/spin/MortonIndex.hpp

```diff
@@ -548,6 +513,7 @@ template <typename CoordType>
 struct PointHash
 {
   using MortonIndex = std::size_t;
+  using result_type = std::size_t;
```

Comment:
Note: This was the only other change to make `axom::FlatMap` work with the `PointHash` class!
(i.e. adding an expected typedef)